code/modules/clothing/under/badges.dm

```diff
@@ -0,0 +1,43 @@
+/// Badges: Like an accessory for your armor vest
+/obj/item/badge
+	name = "badge"
+	desc = "A badge representing... something."
+	icon = 'icons/obj/clothing/badges.dmi'
+	icon_state = "badge"
+	item_state = ""
+	mob_overlay_icon = 'icons/mob/clothing/badges.dmi'
+	w_class = WEIGHT_CLASS_TINY
+	var/owner_string
+
+/obj/item/badge/proc/try_attach(obj/item/clothing/suit/suit_target, mob/user)
+	if(suit_target.attached_badge)
+		to_chat(user, span_warning("There's already a badge on \the [suit_target]."))
```

Comment:
Balloon alert instead?

Reply:
`/obj/item/clothing/accessory` does not use a balloon alert so it would be jarring to be different when it is (user-facing) such a similar item